src/features/MovieDetails/MovieInfo/index.js

```diff
@@ -0,0 +1,48 @@
+import {
+  MovieDetailsWrapper,
+  MovieDetailsImage,
+  MovieDetailsTitle,
+  MovieDetailsSubtitle,
+  MovieDetailsInfo,
+  MovieDetailsExtraInfo,
+  MovieDetailsExtraInfoLabel,
+  MovieDetailsDescription,
+} from "./styled";
+import { MovieTags } from "../../../components/MovieTile/MovieTags";
+import { MovieRating } from "../../../components/MovieTile/MovieRating";
+
+export const MovieInfo = ({
+  poster,
+  title,
+  subtitle,
+  production,
+  releaseDate,
+  tags,
+  rating,
+  votes,
+  description,
+}) => (
+  <>
+    <MovieDetailsWrapper>
+      <MovieDetailsImage
+        src={"https://image.tmdb.org/t/p/w342" + poster}
+        alt={`${title} movie poster`}
+      />
+      <MovieDetailsInfo>
+        <MovieDetailsTitle>{title}</MovieDetailsTitle>
+        <MovieDetailsSubtitle>{subtitle}</MovieDetailsSubtitle>
+        <MovieDetailsExtraInfo>
+          <MovieDetailsExtraInfoLabel>Production:</MovieDetailsExtraInfoLabel>{" "}
+          {production}
+        </MovieDetailsExtraInfo>
+        <MovieDetailsExtraInfo>
+          <MovieDetailsExtraInfoLabel>Release date:</MovieDetailsExtraInfoLabel>{" "}
+          {releaseDate ? releaseDate.replace(/-/g, ".") : ""}
+        </MovieDetailsExtraInfo>
+        <MovieTags tags={tags ? tags.map((tag) => tag.name) : ""} />
+        <MovieRating big="true" rating={rating} votes={votes} />
```

Comment:
The styling of this block does not match the design in Figma. The movie rating should have a larger size and the number of votes should have a different color.

Reply:
You are right. That was the last element i've been working on and probably overlooked it. I'm happy you notced that and I will change it in a second